rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.stats/13_fields.yml

```diff
@@ -4,13 +4,12 @@ setup:
   - do:
       indices.create:
           index:  test1
-          wait_for_active_shards: all
           body:
               settings:
                 # Limit the number of shards so that shards are unlikely
                 # to be relocated or being initialized between the test
                 # set up and the test execution
-                index.number_of_shards: 3
+                index.number_of_shards: 2
```

Comment:
Nit: do we even need to set this?

Reply:
Many of these stats are per shard. I think the intention is to verify that these tests work with multiple shards. If it works with multiple shards it also works when indices stats targets an index with a single shard.